src/antsibull_changelog/yaml.py

```diff
@@ -42,7 +42,7 @@ def load_yaml(path: str) -> Any:
         return yaml.load(stream, Loader=_SafeLoader)
 
 
-def store_yaml(path: str, content: Any, nice: bool = False) -> None:
+def store_yaml(path: str, content: Any, nice: bool = False, sort: bool = False) -> None:
```

Comment:
This argument name isn't very useful. Setting `sort=False` actually **enables** sorting. That's very counter-intuitive.

Reply:
Good point, we are disabling the alphabetical sorting of pyyaml which is enabled by default to get the correct version sorting, I'll change this to be more descriptive.